scripts/copy-packages-tgz.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+# Usage: copy-packages-tgz.sh <SRCDIR>
+#
+# This script copies package tarballs generated from the $SRCDIR workspace
```

Comment:
Why do we need to copy the tarballs at all? Aka can't the resolutions target the original package files?